Makefile

```diff
@@ -192,6 +192,13 @@ compile-documentation-scripts:
 compile-samples:
 	${ROOTDIR}/node_modules/.bin/tsc --project ./documentation/samples/tsconfig.json
 
+.PHONY: validate-samples
+validate-samples:
```

Comment:
FWIW here's the similar command we use in the `packs` repo to run this step:

https://github.com/coda/packs/blob/main/Makefile#L123

Up to you if you think it's worth it to use `parallel` here vs just looping.

Reply:
Removed parallel, since it's fast enough now when using the `/dist` version.